stdlib/cache/src/main/ballerina/cache/cache.bal

```diff
@@ -78,9 +78,16 @@ public type Cache object {
         };
         task:Scheduler cacheCleanupTimer = new(cacheCleanupTimerConfiguration);
 
-        _ = cacheCleanupTimer.attach(cacheCleanupService);
-        _ = cacheCleanupTimer.start();
-
+        var attachCacheCleanerResult = cacheCleanupTimer.attach(cacheCleanupService);
+        if (attachCacheCleanerResult is error) {
+            error e = error("Failed to create cache cleanup scheduler.", { message : attachCacheCleanerResult.detail().message });
```

Comment:
This is actually not a `scheduler`, it is a `task`, right? (we just try to schedule the task here)

```suggestion
            error e = error("Failed to create the cache cleanup task.", { message : attachCacheCleanerResult.detail().message });
```